packages/eslint-plugin/package.json

```diff
@@ -25,6 +25,7 @@
   },
   "devDependencies": {
     "@swc/jest": "~0.2.26",
+    "@tanstack/eslint-plugin-query": "~5.28.11",
```

Comment:
For this package, rules is part of the package functionality, so `@tanstack/eslint-plugin-query` needs to be added as `"dependencies"` (instead of `devDependencies`)

Reply:
fixed